Makefile

```diff
@@ -78,6 +78,7 @@ config.gypi: configure
 	fi
 
 install: all
+	-rm -rf '$PREFIX'/lib/node_modules/npm
```

Comment:
This doesn't do what you think it does, the path gets evaluated to `'REFIX'/lib/node_modules/npm`.  I don't want to sound harsh but did you actually test it?